### PR TITLE
Update alt files

### PR DESCRIPTION
**Pull request by stepps00**
Will fix a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/1729.

### CODE REVIEW

**Comment by stepps00** [comment]
This PR, as well as others flagged in whosonfirst-data/whosonfirst-data#1729 add `wof:repo` properties to any alt geometry file that does not currently have the property.

This is important, as a recent change to the `go-whosonfirst-dist` tools require this property in order to create SQLite distributions. cc @nvkelso

File: data/102/062/895/10206289-alt-quattroshapes.geojson [hunk 1]
```diff
@@ -1,16 +0,0 @@
-{
```
Comment (nvkelso) [inline data/102/062/895/10206289-alt-quattroshapes.geojson]: Why is this file deleted?

Comment (stepps00) [reply data/102/062/895/10206289-alt-quattroshapes.geojson]: This file was deleted because it is a duplicate of the actual `data/102/062/895/102062895-alt-quattroshapes.geojson` file. Note the filename of this record does not contain the full `wof:id`. I'm not sure how this was introduced, but it was added in the refector of Switzerland admin records late last year.
